src/Sylius/Behat/Context/Transform/CountryContext.php

```diff
@@ -54,4 +54,12 @@ public function getCountryByName($countryName)
 
         return $country;
     }
+
+    /**
+     * @Transform :countryCode
```

Comment:
```suggestion
     * @Transform :countryName
```

Reply:
Nope, this is a transformer, so I would like to get country code from country name, everything is like it should be 😃